.github/workflows/update-currencies.yml

```
@@ -4,97 +4,71 @@ on:
-              fi
-            else
-              echo "No existing remote branch 'update-currencies'. Skipping rebase."
-            fi
-            if ! git push --set-upstream origin update-currencies; then
-              echo "Second push failed. Please resolve manually."
-              exit 1
-            fi
-          fi
+          # are we looking at an empty JSON array?
+          empty=$(python - << 'PYCODE'
+import json
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 49: could not find expected ':'

(syntax-check)